tests/unit/test_webhook.py

```diff
@@ -513,11 +552,13 @@ def test_patch_acceptable_issue(self, mock_proxy):
             self.assertEqual(rv.status_code, 200)
             self.assertEqual(rv.content_type, 'text/plain')
 
-    @patch('webcompat.webhooks.helpers.private_issue_moderation')
-    def test_patch_acceptable_issue_problem(self, mock_proxy):
+    @patch.object(webcompat.webhooks.model.WebHookIssue,
+                  'moderate_private_issue')
+    @patch.object(webcompat.webhooks.model.WebHookIssue, 'close_private_issue')
+    def test_patch_acceptable_issue_problem(self, mock_close, mock_moderate):
         """Test for accepted issues failed.
 
-        payload: 'ooops'
+        payload: 'oops'
```

Comment:
hehe.

Reply:
After fixing this typo, I realized I had to fix all the tests -- protip: never fix typos.